Marlin/src/gcode/gcode.cpp

```diff
@@ -376,6 +376,11 @@ void GcodeSuite::process_parsed_command(
         case 190: M190(); break;                                  // M190: Wait for bed temperature to reach target
       #endif
 
+      #if ENABLED(HAS_HEATED_CHAMBER)
```

Comment:
This option has not been added to any `Configuration.h` file. Note that the naming should not include `HAS_` for configurable options, but only for items defined in the `Conditionals*` files. And `HAS_` items defined in the `Conditionals*` files should not use `ENABLED()`. I would assume that this feature is enabled by having a temp sensor defined for the heated chamber.

Reply:
Yes, you are correct that is it is enabled by having a temp sensor defined. This is meant to be an additional option on top of that, nested within the temp_chamber option. What do you suggest for a name?

Reply:
See the commits in their current form.